.gitignore

```diff
@@ -0,0 +1,68 @@
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+*.ipch
```

Comment:
That may be useful, but we don't use gcc (clang can create precompiled headers everywhere) and we dont use precompiled headers.